src/modules/transaction/components/TxBroadcasterWithStatus/index.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
```

Comment:
why was this component necessary? is this the generic component?

Reply:
Yes, it is a generic component. It is similar to the abstraction made in store/status -> signAndDispatchTransaction. 
This component could replace UnlockBalanceStatus and RegisterValidatorStatus, but this PR just grows and grows so i decided not to refactor those components